components/GothamistHomepageTopper.vue

```diff
@@ -73,13 +73,14 @@ const latestArticles = computed(() => props.articles.slice(1))
           <div></div>
           <v-card-metadata :article="article" :showComments="false" />
         </v-card>
-        <hr v-if="index < latestArticles.length - 1" class="my-3 block" />
+        <hr class="my-3 block" />
       </div>
       <div class="mb-1">
         <HtlAd
           layout="rectangle"
           slot="htlad-gothamist_index_topper"
           fineprint="Gothamist is funded by sponsors and member donations"
+          fineprintClass="text-center"
```

Comment:
allows us to pass classes to the fineprint text... in this case, to center the text for this particular instance